packages/pds/src/services/account/index.ts

```diff
@@ -145,6 +145,17 @@ export class AccountService {
     await sequenceHandleUpdate(this.db, did, handle)
   }
 
+  async updateEmail(did: string, email: string) {
+    const res = await this.db.db
```

Comment:
Should we do some minimal form of email validation?  Just poked-around quickly at registration and didn't see any, though hmm 🤔